src/styles/main.scss

```diff
@@ -1 +1,68 @@
-// put code here
+@import "animation";
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+}
+
+.container {
+  width: 100%;
+  height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.clock {
+  width: 400px;
+  height: 400px;
+  border-radius: 50%;
+  border: 4px dotted brown;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  &__arrows {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+  }
+
+  &__small-arrow {
+    width: 190px;
+    height: 10px;
+    background-color: green;
+    border-radius: 5px;
+    z-index: 1;
+    position: absolute;
+    top: calc(50vh - 5px);
+    left: calc(50vw - 190px);
+    transform-origin: right;
+    animation: seconds linear 60s infinite;
+  }
+
+  &__big-arrow {
+    width: 100px;
+    height: 16px;
+    background-color: firebrick;
+    border-radius: 10px;
+    z-index: 1;
+    position: absolute;
+    top: calc(50vh - 8px);
+    right: calc(50vw - 100px);
+    transform-origin: left;
+    animation: minutes linear 720s infinite;
```

Comment:
Follow the task description, please 
![image](https://github.com/mate-academy/layout_stop-watch/assets/99403198/8ea5da3c-e132-47f7-884a-ef5636fef8b8)